frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -113,6 +119,13 @@ export const CommandBar = () => {
     useOnClickOutside([searchContainerRef], hideSuggestions);
     useOnBlur(searchContainerRef, hideSuggestions);
 
+    const { features, setTableState } = useGlobalFeatureSearch(3);
+
+    const flags: CommandResultGroupItem[] = features.map((feature) => ({
+        name: feature.name,
+        link: `/projects/${feature.project}/features/${feature.name}`,
```

Comment:
I think there's a route formatter somewhere we might use for this, it's on my mental todo. But it's unlikely to change I suppose